docs/how-to-configure-new-repository.md

```diff
@@ -14,23 +14,21 @@ Documents [Community Membership](../community-membership.md) and
 [CONTRIBUTING](../CONTRIBUTING.md) define how permissions are typically set up
 for the repository.
 
-1. Every repository has two teams associated with it. Typically for the
-   repository `opentelemetry-foo` they will be named `foo-approvers` and
-   `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers` as it
-   always contains subset of people and defines larger scope of privileges.
-2. Even though the members of `foo-maintainers` are included in `foo-approvers`
-   transitively, every member of `foo-maintainers` should be included in
-   `foo-approvers` explicitly, and with the "Maintainer" GitHub privileges. So
-   repository maintainers can invite new approvers to the team.
-3. The team `foo-approvers` has `Write` permissions for the repository.
-4. The team `foo-maintainers` has `Maintain (beta)` permissions for the
+1. Every repository has three teams associated with it. Typically for the
+   repository `opentelemetry-foo` they will be named `foo-triagers`, `foo-approvers`,
```

Comment:
should `foo-triagers` be a parent of `foo-approvers`. This will ensure approvers will be notified when triagers roup is mentioned in a comment.

Reply:
Ah, I see why you nest them. Yes, I agree that this would make sense; approvers and maintainers should also be performing triage duty.